translations/de.ts

```diff
@@ -1762,7 +1618,7 @@ Profile does not contain your history.</source>
     <message>
         <location filename="../src/loginscreen.ui" line="757"/>
         <source>If the profile does not have a password, qTox can skip the login screen</source>
-        <translation>Für Profile ohne Passwort kann das Login Fenster übersprungen werden</translation>
+        <translation>Für Profile ohne Passwort kann das Login-Fenster übersprungen werden</translation>
```

Comment:
Well ... it's a dot. So ... make a decision and stick with it. I personally prefer correctness. Technically it makes no difference. If source is corrected, you need to re-translate (or tell Qt: use my correct sentence).